gene/mutate.go

```diff
@@ -69,8 +74,10 @@ func PointMutation(g *Gene, genomePos int, alt dna.Base) (EffectPrediction, erro
 		} else {
 			if answer.AaAlt[0] == dna.Stop {
 				answer.Consequence = Nonsense
+				answer.StopDist = 0
 			} else if answer.AaRef[0] == dna.Stop {
 				answer.Consequence = DisruptStop
+				//TODO calculate dist to new stop
 			} else if answer.AaPos == 0 {
 				answer.Consequence = DisruptStart
```

Comment:
Yes, that seems to be the consensus. The trick is that the ATG does not necessarily need to be in-frame so this can result in a frameshift. Do you think I should try to predict the new start and update the cdna accordingly?